code/modules/clothing/head/misc.dm

```diff
@@ -259,4 +259,15 @@
 	name = "paper sack hat"
 	desc = "A paper sack with crude holes cut out for eyes and a sketchy smile drawn on the front. Not creepy at all."
 	icon_state = "papersack_smile"
-	flags_inv = HIDEHAIR|HIDEFACE|HIDEEARS
\ No newline at end of file
+	flags_inv = HIDEHAIR|HIDEFACE|HIDEEARS
+
+/obj/item/clothing/head/crown
+	name = "crown"
+	desc = "A crown fit for a king, a petty king maybe."
+	icon_state = "crown"
+	armor = list(melee = 50, bullet = 0, laser = 0,energy = 20, bomb = 0, bio = 0, rad = 0) //for a warrior king //energy cause gold is reflective //worse than sec helm overall

```

Comment:
That can already happen, what concerns me is that lots of little items like this creep into the game, everyone sees that so much stuff has near the same armour counts as actual armour, so some bright chap gets it in their head to come along and double all the sec armour counts to make them stand out.

then the cycle continues in the eternal buff and any attempt we make to nerf it just leads to a big slap fight and please I dont' want to go through that again just make it stop make the pain stop I can'ts leep I'm getting ulcers please it's easier to buff things than it is to nerf them always start out by making the item weak and then buffing it later based on feedback.